### PR TITLE
LOG-2751: Using related images naming scheme

### PR DESCRIPTION
**Pull request by Red-GV**
This PR adds the images under the `relatedImages` section. 

/cc @syedriko 
/assign @jcantrill 

### Links
- JIRA: https://issues.redhat.com/browse/LOG-2751

### CODE REVIEW

**Comment by vimalk78** [comment]
need change the variable name in Makefile as well?
https://github.com/openshift/cluster-logging-operator/blob/576cf90d497744bc53bdab2f2efe7aeebb068583/Makefile#L195

**Comment by Red-GV** [comment]
I do not believe so, @vimalk78. Only the name of the image had changed, not the image's source.

**Comment by Red-GV** [comment]
Actually, I think you're right, @vimalk78. I didn't realize exactly how the test framework worked.

File: bundle/manifests/logging.openshift.io_clusterloggings.yaml [hunk 1]
```diff
@@ -502,7 +502,9 @@ spec:
                         nullable: true
                         properties:
                           diskThresholdPercent:
-                            description: The threshold percentage of ES disk usage that when reached, old indices should be deleted (e.g. 75)
+                            description: The threshold percentage of ES disk usage
```
Comment (vparfonov) [inline bundle/manifests/logging.openshift.io_clusterloggings.yaml:505]: @Red-GV  Looks like needless changes, probably related to your OS. (I have same with my MacOS)

**Pull request merged**